source/js/buyers-guide/product-tab.js

```diff
@@ -0,0 +1,33 @@
+export default () => {
+  document.querySelectorAll(`#product-tab`).forEach((element) => {
+    const allTabs = element.querySelectorAll("span[data-product-label]");
+
+    // First tab is active by default
+    allTabs[0].classList.remove("tw-bg-white", "tw-text-black");
+    allTabs[0].classList.add("tw-bg-black", "tw-text-white");
+
+    allTabs.forEach((tabButton) => {
+      tabButton.addEventListener("click", (event) => {
+        event.stopPropagation();
+        event.preventDefault();
+        // reset active tab stylings
+        for (var i = 0; i < allTabs.length; i++) {
+          allTabs[i].classList.remove("tw-bg-black", "tw-text-white");
+          allTabs[i].classList.add("tw-bg-white", "tw-text-black");
+        }
+
+        // set active tab styling
+        event.currentTarget.classList.remove("tw-bg-white", "tw-text-black");
+        event.currentTarget.classList.add("tw-bg-black", "tw-text-white");
+
+        // set active tab content
+        const tabIndex = parseInt(
+          event.currentTarget.getAttribute("data-product-label")
+        );
+        document.getElementById(
+          "product-tab-content"
+        ).style.transform = `translateX(${tabIndex * -100}%)`;
```

Comment:
Just thinking in terms of separation of technolgies: let's make this a CSS-var controlled rule for `#product-tab-content`, e.g.
```CSS
#product-tab-content {
  --x-offset: 0;
  transform: translateX(var(--x-offset));
```
And then here on the JS side instead use
```JS
document.getElementById(`product-tab-content`).style.setProperty(`--x-offset`, `${tabIndex * -100}%`);
```